taxcalc/records.py

```diff
@@ -386,6 +392,23 @@ def _adjust(self, year):
             adj_array = self.ADJ['INT{}'.format(year)][self.agi_bin].values
             self.e00300 *= adj_array
 
+    def _extrapolate_benefits(self, year):
+        """
+        Extrapolate benefit variables
+        """
+        ben = self.BEN['ssi_benefits_{}'.format(year)][self.RECID].values
+        self.ssi_ben = np.nan_to_num(ben)
+        ben = self.BEN['snap_benefits_{}'.format(year)][self.RECID].values
+        self.snap_ben = np.nan_to_num(ben)
+        ben = self.BEN['vb_benefits_{}'.format(year)][self.RECID].values
+        self.vet_ben = np.nan_to_num(ben)
+        ben = self.BEN['ss_benefits_{}'.format(year)][self.RECID].values
+        self.ss_ben = np.nan_to_num(ben)
+        ben = self.BEN['medicare_benefits_{}'.format(year)][self.RECID].values
+        self.mcare_ben = np.nan_to_num(ben)
+        ben = self.BEN['medicaid_benefits_{}'.format(year)][self.RECID].values
+        self.mcaid_ben = np.nan_to_num(ben)
+
```

Comment:
@andersonfrailey do you think that we should do apply the `np.nan_to_num` function in the `taxdata` repo?

Reply:
No because `NaN` values don't occur until we're assigning benefits. This line of code matches benefits by RECID:
```python
ben = self.BEN['medicaid_benefits_{}'.format(year)][self.RECID].values
```

Because we drop all of the records without benefits when we create the extrapolated benefits file, those who have no benefits receive a `NaN` value instead and so we have to apply the function here.

Reply:
Ok, that makes sense to me. Thanks @andersonfrailey